### PR TITLE
Feature/bolt money

### PR DESCRIPTION
**Pull request by vitalie-ucommerce**
Use the new `ApiMoney` class everywhere where price is shown

### CODE REVIEW

File: src/Ucommerce.Avenue.Umbraco/Api/AvenueClothingApiBasketController.cs [hunk 1]
```diff
@@ -98,7 +91,7 @@ public IHttpActionResult GetBasket()
                 var product = CatalogLibrary.GetProduct(line.Sku);
                 var url = UrlService.GetUrl(CatalogContext.CurrentCatalog, new[] {product});
                 var imageUrl = product.PrimaryImageUrl;
-                var lineTotal = new Money(line.Total.Value, currency);
+                var lineTotal = new ApiMoney(line.Total.Value, CatalogContext.CurrentPriceGroup.CurrencyISOCode);
```
Comment (ilny) [inline src/Ucommerce.Avenue.Umbraco/Api/AvenueClothingApiBasketController.cs]: `var currencyISOCode = CatalogContext.CurrentPriceGroup.CurrencyISOCode` and use it in all the cases.

File: src/Ucommerce.Avenue.Umbraco/Api/AvenueClothingApiProductController.cs [hunk 2]
```diff
@@ -131,12 +130,12 @@ private PriceViewModel GetPriceViewModel(decimal priceExclTax, decimal priceIncl
...
         }
 
-        private MoneyViewModel GetMoneyViewModel(decimal price, Currency currency)
+        private MoneyViewModel GetMoneyViewModel(decimal price, string currency)
         {
             return new MoneyViewModel()
             {
                 Value = price,
-                DisplayValue = new Money(price, currency).ToString()
+                DisplayValue = new ApiMoney(price, currency).ToString()
```
Comment (ilny) [inline src/Ucommerce.Avenue.Umbraco/Api/AvenueClothingApiProductController.cs]: What's the reason for keeping `currency` as variable name here and not in the previous cases?
Let's have it consistent!

File: src/Ucommerce.Avenue.Umbraco/Api/AvenueClothingApiBasketController.cs (hunk 1 above)
Comment (vitalie-ucommerce) [reply src/Ucommerce.Avenue.Umbraco/Api/AvenueClothingApiBasketController.cs]: Agree, 5 use cases replaced

File: src/Ucommerce.Avenue.Umbraco/Api/AvenueClothingApiProductController.cs (hunk 2 above)
Comment (vitalie-ucommerce) [reply src/Ucommerce.Avenue.Umbraco/Api/AvenueClothingApiProductController.cs]: This is in a separate method, but I agree with you 👍

**Pull request merged**